src/current/v25.2/detect-hotspots.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Detect Hotspots
+summary: Learn how to detect hotspots using real-time monitoring and historical logs in CockroachDB.
+toc: true
+---
+
+This page provides practical guidance for identifying common [hotspots]({% link {{ page.version.version }}/understand-hotspots.md %}) in CockroachDB clusters using real-time monitoring and historical logs.
+
+## Before you begin
+
+- Review the [Understand hotspots page]({% link {{ page.version.version }}/understand-hotspots.md %}) for definitions and concepts.
+- Ensure you have access to the DB Console Metrics and the relevant logs.
+- Confirm that you have the necessary permissions to modify the application or schema.
```

Comment:
This might be superfluous, it might not necessarily be the responsibility of the person investigating the issue to also fix it.